### ISSUE TITLE
Suggestion to use Jekyll

### ISSUE THREAD

**Issue by EdricChan03** [MartonDev/Material-Design-18#6]
Using [Jekyll](https://jekyllrb.com) on this project site could help to reduce the amount of boilerplate code in every HTML file.

**Comment by Richienb** [comment]
Right. I have a bit of experience using Jekyll and I might be able to help @MartonDev out by introducing it.
